runtime/tests/port/j9strTest.c

```diff
@@ -1534,40 +1534,45 @@ j9str_WinacpToMutf8(struct J9PortLibrary *portLibrary)
  * @param[in] buffer - pointer to the buffer intended for atoe_vsnprintf()
  * @param[in] bufferLength - the buffer length
  * @param[in] expectedOutput - The expected string
+ * @param[in] expectedCount - The expected response from atoe_vsnprintf()
  * @param[in] format - The string format
  *
  * @return TEST_PASS on success
  *         TEST_FAIL on error
  */
 static int
-printToBuffer(struct J9PortLibrary *portLibrary, BOOLEAN precision, char *buffer, size_t bufferLength, const char *expectedOutput, const char *format, ...)
+printToBuffer(struct J9PortLibrary *portLibrary, BOOLEAN precision, char *buffer, size_t bufferLength, const char *expectedOutput, int expectedCount, const char *format, ...)
 {
 	PORT_ACCESS_FROM_PORT(portLibrary);
-	int stringLength = 0;
+	int stringLength, expectedLength;
+	int actualCount;
 	va_list args;
 
 	va_start(args, format);
-	stringLength = atoe_vsnprintf(buffer, bufferLength, format, args);
+	actualCount = atoe_vsnprintf(buffer, bufferLength, format, args);
+	expectedLength  = expectedOutput == NULL ? 0 : strlen(expectedOutput);
+	stringLength = buffer == NULL ? 0 : strlen(buffer);
```

Comment:
Please put constants on the left of `==`, collapse the double space, and use parentheses:
```c
	expectedLength = (NULL == expectedOutput) ? 0 : strlen(expectedOutput);
	stringLength = (NULL == buffer) ? 0 : strlen(buffer);
	va_end(args);
```
And while you're here, please remove the space after `va_end`.